src/AppBundle/Resources/views/Admin/Client/Client/details.html.twig

```diff
@@ -24,11 +24,53 @@
 
 {% block pageContent %}
 
-    <p class="flush--bottom">{{ 'client' | trans({}, 'common') }}: <span class="bold">{{ client.fullName }}</span></p>
+<dl class="govuk-summary-list">
+    <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+            {{ 'client' | trans({}, 'common') }}
+        </dt>
+        <dd class="govuk-summary-list__value">
+            {{ client.fullName }}
+        </dd>
+    </div>
     {% if client.archivedAt %}
-        <p class="flush--bottom">Archived on: <span class="bold">{{ client.archivedAt | date("j M Y") }}</span></p>
+    <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+            {{ 'archivedOn' | trans({}, 'common') }}
+        </dt>
+        <dd class="govuk-summary-list__value">
+            {{ client.archivedAt | date("j M Y") }}
+        </dd>
+    </div>
+    {% endif %}
+
+    <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+            {{ 'courtOrderNumber' | trans({}, 'common') }}
+        </dt>
+        <dd class="govuk-summary-list__value">
+            {{ client.caseNumber }}
+        </dd>
+    </div>
+    {% if client.deletedAt %}
+    <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+            {{ 'dischargedOn' | trans({}, 'common') }}
+        </dt>
+        <dd class="govuk-summary-list__value behat-region-discharged-on">
+            {{ client.deletedAt | date("j M Y") }}
+        </dd>
+    </div>
     {% endif %}
-    <p>{{ 'courtOrderNumber' | trans({}, 'common') }}: <span class="bold">{{ client.caseNumber }}</span></p>
+</dl>
+    {#<p class="flush--bottom">{{ 'client' | trans({}, 'common') }}: <span class="bold">{{ client.fullName }}</span></p>#}
+    {#{% if client.archivedAt %}#}
+        {#<p class="flush--bottom">Archived on: <span class="bold">{{ client.archivedAt | date("j M Y") }}</span></p>#}
+    {#{% endif %}#}
+    {#<p>{{ 'courtOrderNumber' | trans({}, 'common') }}: <span class="bold">{{ client.caseNumber }}</span></p>#}
+    {#{% if client.deletedAt %}#}
+        {#<p class="flush--bottom behat-region-discharged-on">Discharged on: <span class="bold">{{ client.deletedAt | date("j M Y") }}</span></p>#}
+    {#{% endif %}#}
```

Comment:
Should probably ditch this commented out code, not a blocker though

Reply:
Revised to show a summary as per previous review. Should not have been commented